### PR TITLE
Use Debian instead of Ubuntu for Travis and build the Debian package

### PR DESCRIPTION
**Pull request by radare**
**Detailed description**

We need to build things on Debian distros, not ubuntu ones for portability reasons. i pick this one because its the same we are using in radare

But also we want to have the .deb files uploaded and not having to depend on the version of r2.

**Test plan**

Ideally it should use Gitlab CI

**Closing issues**
none

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ branches:
     - master
 
 os: linux
-dist: bionic
+dist: trusty
```
Comment (thestr4ng3r) [inline .travis.yml:9]: trusty is not debian

**Comment by radare** [comment]
Stretch i guess. What a shitty names

**Review by ITAYC0HEN — APPROVED** [review]
Looks good :)
I don't understand the reasons behind it, but sure they are legit.

**Comment by radare** [comment]
To get the upload working will require to configure the dist keys in gh settings.

The reason to use debian is for consistency and portability. Ubuntu is a superset of debian. Ubuntu builds dont need to run on debian. All deb files are compiled on debian distros

**Comment by radare** [comment]
Ideally we should use github actions instead of travis here.

**Comment by thestr4ng3r** [comment]
Still broken though

**Comment by thestr4ng3r** [comment]
If you still want to do this, you can add another github workflow similar to the jessie one and just run it in whatever docker base image you want.

**Pull request closed without merging**